docs/troubleshooting/detections/detection-rules.asciidoc

```diff
@@ -3,6 +3,31 @@
 
 This topic covers common troubleshooting issues when creating or managing <<rules-ui-create, detection rules>>.
 
+[discrete]
+[[ML-rules-ts]]
+=== {ml-cap} rules
+
+[discrete]
+[[start-ML-jobs-ts]]
+.{ml-cap} rule is failing and a required {ml} job is stopped
+[%collapsible]
+====
+If a {ml} rule is failing, check to make sure the required {ml} jobs are running and start any jobs that have stopped.
+
+. Go to **Detect** -> **Rules**, then select the {ml} rule. The required {ml} jobs and their statuses are listed in the Definition section.
++
+[role="screenshot"]
+image::images/rules-ts-ml-job-stopped.png[]
++
+. If a required {ml} job isn't running, select **ML job settings** in the upper-right corner of the page, then search for the {ml} job.
+. Turn on the **Run job** switch for the required {ml} job.
++
+[role="screenshot"]
+image::images/rules-ts-start-ml-job.png[]
++
+. Re-run the {ml} detection rule.
```

Comment:
Not sure if the wording for this is in our style guide(s) anyway, but I think typically Elastic doesn't use hyphens too often. Let me know if you find out otherwise. 

```suggestion
. Rerun the {ml} detection rule.
```